packages/ember-model/tests/belongs_to_test.js

```diff
@@ -335,7 +335,7 @@ test("should be able to set nonembedded relationship to null", function() {
   });
 
   equal(post.get('author'), null);
-  deepEqual(post.toJSON(), {id: 1, author_id: null});
+  deepEqual(post.toJSON(), {id: 1, author_id: undefined});
```

Comment:
@eventualbuddha I'm not sure what change in Ember resulted in this change. I'll track it down